core/src/main/java/hudson/cli/InstallPluginCommand.java

```diff
@@ -83,7 +83,7 @@ public String getShortDescription() {
     @Override
     protected int run() throws Exception {
         Jenkins h = Jenkins.get();
-        h.checkPermission(PluginManager.UPLOAD_PLUGINS);
+        h.checkPermission(Jenkins.ADMINISTER);
```

Comment:
maybe we can call Jenkins j at least (or the full word jenkins), or oneline like :1st_place_medal: 
`Jenkins.get().checkPermission(Jenkins.ADMINISTER); `?

Reply:
that makes sense.